contracts/HubPool.sol

```diff
@@ -549,64 +549,70 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
      * relay roots to the SpokePool on the network specified in the leaf.
      * @dev In some cases, will instruct spokePool to send funds back to L1.
      * @notice Deletes the published root bundle if this is the last leaf to be executed in the root bundle.
-     * @param poolRebalanceLeaf Contains all data neccessary to reconstruct leaf contained in root bundle and to
-     * bridge tokens to HubPool. This data structure is explained in detail in the HubPoolInterface.
+     * @param chainId ChainId number of the target spoke pool on which the bundle is executed.
+     * @param bundleLpFees Array representing the total LP fee amount per token in this bundle for all bundled relays.
+     * @param netSendAmounts Array representing the amount of tokens to send to the SpokePool on the target chainId.
+     * @param runningBalances Array used to track any unsent tokens that are not included in the netSendAmounts.
+     * @param leafId Index of this executed leaf within the poolRebalance tree.
+     * @param l1Tokens Array of all the tokens associated with the bundleLpFees, nedSendAmounts and runningBalances.
      * @param proof Inclusion proof for this leaf in pool rebalance root in root bundle.
      */
-    function executeRootBundle(PoolRebalanceLeaf memory poolRebalanceLeaf, bytes32[] memory proof) public nonReentrant {
+    function executeRootBundle(
```

Comment:
This is the primary change of this PR: spread the `PoolRebalanceLeaf` object to improve etherscan tx processing.